### ISSUE TITLE
Customized new_auto_mod() not bringing likes

### ISSUE THREAD

**Issue by inamics** [instabot-py/instabot.py#140]
Hello together,

due to the fact that my hosting provider has a max_execution_time of 30 seconds on scripts I have implemented a Cronjob that start the script all 5 minutes. Because of that I changed the code a little bit to stay below the 30 seconds (10 times over 30 seconds and the cronjob will be deleted).

The only thing I changed is actually the "while true" loop in the new_auto_mod_like() function. So ofcourse I know the function new_auto_mod_like() will be executed only once but nevertheless in genereal the script doesn't like pictures at all. Sometimes it likes one or at max two pictures but in 90 percent it likes 0.

Can somebody tell me how to increase that? Its okay for me to just follow or comment 1 user/picture but I definitely need more likes to be executed.

Thanks in advance

**Comment by LevPasha** [comment]
You relogin every 30 seconds? It is not very good i think. Change your provider)

**Comment by HenryCooperGithub** [comment]
**Please update to the most recent version of instabot.py**

If your problem still persists open a new issue and attach the following information:

- Your bot configuration (parameters)
- Full console output
